packages/compiler-cli/src/ngtsc/path/src/logical.ts

```diff
@@ -95,3 +95,8 @@ export class LogicalFileSystem {
     return this.cache.get(physicalFile) !;
   }
 }
+
+function createLogicalProjectPath(file: string, rootDir: string): LogicalProjectPath {
```

Comment:
Let's make this a private method of the `LogicalFileSystem` instead. Also:

```suggestion
function createLogicalProjectPath(file: AbsoluteFsPath, rootDir: AbsoluteFsPath): LogicalProjectPath {
```